### PR TITLE
Brick walls sprite

### PR DESCRIPTION
**Pull request by AlphaBuffalo**
I would try to code this in, but I'm sure I would end up breaking more shit than needed

### CODE REVIEW

**Comment by ghost** [comment]
Pics

**Comment by AlphaBuffalo** [comment]
abb932ac593fccaaec3b15e25b34d1eb

**Comment by AlphaBuffalo** [comment]
resprited the mineral walls

**Comment by AlphaBuffalo** [comment]
silver sheet
Bricks are fun

File: code/game/turfs/simulated/walls_mineral.dm [hunk 1]
```diff
@@ -13,6 +13,13 @@
 	walltype = "wood"
 	mineral = "wood"
 
+/turf/simulated/wall/mineral/brick
+	name = "Brick Wall"
+	desc = "A wall with brick siding, it looks nice"
+	icon_state = "Brick 0"
```
Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: code/game/turfs/simulated/walls_mineral.dm [hunk 2]
```diff
@@ -13,6 +13,13 @@
 	walltype = "wood"
 	mineral = "wood"
 
+/turf/simulated/wall/mineral/brick
+	name = "Brick Wall"
```
Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: "brick wall" is the naming convention

**Comment by AutismSpider** [comment]
>Brick only has throw-force of five!

Make it 10 or some thing its a bloody brick

**Comment by AlphaBuffalo** [comment]
@DrCelt idk what that code does I just copy paste and changed the name

**Comment by gbasood** [comment]
@AlphaBuffalo Lower-case for improper nouns regardless of where they are in a sentence, so we can use textual helpers like \the for grammar.

**Comment by clusterfack** [comment]
What did you mean by

>resprited the mineral walls


All of them or what?

**Comment by AlphaBuffalo** [comment]
no I just copied the silver wall, recolored, and renamed it

File: code/game/turfs/simulated/walls_mineral.dm [hunk 3]
```diff
@@ -13,6 +13,13 @@
 	walltype = "wood"
 	mineral = "wood"
 
+/turf/simulated/wall/mineral/brick
+	name = "brick wall"
+	desc = "A wall with brick siding, it looks nice"
+	icon_state = "Brick 0"
```
Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: this should probably be brick0?

**Comment by clusterfack** [comment]
Make sure this works and everything then let me know

File: code/game/turfs/simulated/walls_mineral.dm [hunk 4]
```diff
@@ -13,6 +13,13 @@
 	walltype = "wood"
 	mineral = "wood"
 
+/turf/simulated/wall/mineral/brick
+	name = "brick wall"
+	desc = "A wall with brick siding, it looks nice"
+	icon_state = "Brick0"
```
Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: Case sensitive brick0. Did you test this?

Comment (AlphaBuffalo) [inline code/game/turfs/simulated/walls_mineral.dm]: Yeah, tried placing the walls in build mode and the sprites werent working, is this why?

Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: Probably.

Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: Are the icon states of the brick right in the .dmi?

Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: They need to be brick8 etc

Comment (AlphaBuffalo) [inline code/game/turfs/simulated/walls_mineral.dm]: They are, but they're out of order because I haven't got around to fixing them.

Comment (ghost) [inline code/game/turfs/simulated/walls_mineral.dm]: Nerd

**Comment by AlphaBuffalo** [comment]
7e4ab521660b6fd191cfba035a8c5588

Proof the sprites work, and smoothing properly.

**Comment by CptWad** [comment]
:+1:

**Comment by AlphaBuffalo** [comment]
@6789 I hope I referenced the right issue

**Comment by AlphaBuffalo** [comment]
This is also for the snow map #6789

**Comment by AlphaBuffalo** [comment]
@clusterfack Its ready to merge

**Pull request merged**